packages/bot-web-ui/src/components/dashboad/index.js

```diff
@@ -0,0 +1,4 @@
+import Dashboard from './dashboard.jsx';
```

Comment:
As we start a new project I suggest using typescript

Reply:
The configuration is not ready yett. I will change this as it gets ready.